src/app/core/component/comment-list/comment-list.component.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('CommentListComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
-  });
+  // it('should create', () => {
```

Comment:
So you want me to delete every *.spec.ts file in this project?

Reply:
No, I was asking you to delete just this test class since it does not test anything 
we do have tests in .spec files (i admit that there are only a few)

Reply:
Done